tests/manual/rl-iosched.py

```diff
@@ -198,7 +198,7 @@ def add_test(name, fn):
 
 def mixed_show_stat_header():
     print('-' * 20 + '8<' + '-' * 20)
-    print('name           througput(kbs) iops lat95(us) queue-time(us) execution-time(us) K(bw) K(iops) K()')
+    print('name           throughput(kbs) iops lat95(us) queue-time(us) execution-time(us) K(bw) K(iops) K()')
```

Comment:
This appears to be a title of a formatted output, so somebody should check it doesn't break the formatting. Or simply leave this typo as-is - it's just a test anyway...